### PR TITLE
Ssd mitigation changes

### PR DESCRIPTION
**Pull request by rkdevi27**
**- What I did**
I have added the ssd mitigation changes which collects ssd logs for S6100 with 3IE3 ssd.
**This Pull request has dependency on the pull request raised in sonic-utilities branch
https://github.com/Azure/sonic-utilities/pull/829** 

**- How I did it**
I have added the following changes:
Collect ssd logs before fast-reboot.
Added after removing a sonic image and before installing an **image.**
Block the fast-reboot for 2 minutes from the time of sonic image installation.
**- How to verify it**
I have tested the functionality and attaching the logs here.
[console_logs.txt](https://github.com/Azure/sonic-buildimage/files/4282108/console_logs.txt)

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/image_config/fstrim/fstrim.service [hunk 1]
```diff
@@ -3,4 +3,4 @@ Description=Discard unused blocks
 
 [Service]
 Type=oneshot
-ExecStart=/sbin/fstrim -av
+ExecStart=/usr/bin/sonic-fstrim
```
Comment (jleveque) [inline files/image_config/fstrim/fstrim.service]: Where does `sonic-fstrim` get installed?

Comment (rkdevi27) [reply files/image_config/fstrim/fstrim.service]: This has been included in sonic-utilities/scripts as part of pull request 
https://github.com/Azure/sonic-utilities/pull/829 

As mentioned, the PR 4214 has dependency on sonic-utilities pull request #829.

**Comment by rkdevi27** [comment]
I have reverted the fstrim changes and 2 minutes delay changes and updated the PR.

File: files/image_config/fstrim/fstrim.service (hunk 1 above)
Comment (sujinmkang) [inline files/image_config/fstrim/fstrim.service]: can we run sonic-fstrim when it's available?

Comment (rkdevi27) [reply files/image_config/fstrim/fstrim.service]: Yes, we can. I have changed the permissions in the code base itself and built the image and tested it.

**Comment by rkdevi27** [comment]
Retest vsimage please

File: files/image_config/fstrim/fstrim.service (hunk 1 above)
Comment (sujinmkang) [reply files/image_config/fstrim/fstrim.service:6]: @rkdevi27 can we make platform_log_ssd_health more generically and run it from fstrim.service? And then we don't need sonic-fstrim any more.

Comment (rkdevi27) [reply files/image_config/fstrim/fstrim.service]: Sujin, 
As suggested, I have changed log_ssd_health as generic.

**Comment by rkdevi27** [comment]
retest vsimage please

**Comment by sujinmkang** [comment]
retest vsimage please

**Comment by sujinmkang** [comment]
Retest vsimage please

**Pull request merged**